src/QueryFilters.php

```diff
@@ -59,6 +62,6 @@ public function apply(Builder $builder)
      */
     public function filters()
     {
-        return $this->request->all();
+        return is_array($this->request) ? $this->request : (method_exists($this->request, 'all') ? $this->request->all() : []);
```

Comment:
Instead of putting everything in one line with two ternary operators can we split the first part ? I think something like this would be more readable instead of understanding a long line with two ternaries

```php
public function filters()
{
    if (is_array($this->request)) {
        return $this->request;
    }

    return method_exists($this->request, 'all') ? $this->request->all() : [];
}
```